packages/kustomize.yml

```diff
@@ -0,0 +1,4 @@
+version: 3.5.1
+upperLimit: 3.5.4
```

Comment:
let's make this 3.6.0. If kustomize does semantic versioning right, there should be definitely no issues up to 3.6 and it means we don't have to constantly update.